.travis.yml

```diff
@@ -24,8 +24,8 @@ jobs:
       cache:
         cargo: true
       script:
-        - cargo check
-        - cargo test
+        - cargo check --frozen
```

Comment:
actually, i think this will fail unless we do `cargo fetch --locked` in another stage

Reply:
I think there needs to be an
```yaml
install:
  - cargo fetch --locked
```